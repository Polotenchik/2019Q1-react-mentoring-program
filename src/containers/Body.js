import React from 'react';
import { Header } from './Header';
import { Content } from './Content';
import { Footer } from '../components/Footer';


export const Body = (props) => {
    return (
        <>
            <div className="wrapper">
                <Header headerType={ props.headerType } />
                <Content contentType={ props.contentType } headerType={ props.headerType } />
            </div>
            <Footer />
        </>
    );
}