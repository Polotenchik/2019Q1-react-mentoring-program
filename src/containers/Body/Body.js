import React from 'react';
import { Header, Content,  } from '../index';
import { Footer } from '../../components';

export const Body = (props) => {
    return (
        <>
            <div className="wrapper">
                <Header 
                    headerType={ props.headerType } 
                    id={ props.movie }
                />
                <Content 
                    headerType={ props.headerType } 
                    movies={  props.movies }
                    handler = { props.getMovieId } 
                />
            </div>
            <Footer />
        </>
    );
}