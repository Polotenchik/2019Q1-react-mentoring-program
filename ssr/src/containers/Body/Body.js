import React from 'react';
import {
  BrowserRouter as Router, Route, Link, Switch, Redirect,
} from 'react-router-dom';
import Header from '../Header/Header';
import Content from '../Content/Content';
import { NoResultsBlock } from '../../components';
import { Footer } from '../../components';

export const Body = () => (
        <>
            <div className="wrapper">
                <Header/>
                <Switch>
                    <Route exact path='/' component={ NoResultsBlock } />
                    <Route path='/search' component={ Content } />
                </Switch>
            </div>
            <Footer />
        </>
);
