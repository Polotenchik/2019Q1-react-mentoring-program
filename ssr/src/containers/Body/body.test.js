import React from 'react';
import { Body } from './Body';
import { shallow } from 'enzyme';

describe('<Body />', () => {
  describe('Rendering', () => {
    it('should render the Body', () => {
      const wrapper = shallow(<Body />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});