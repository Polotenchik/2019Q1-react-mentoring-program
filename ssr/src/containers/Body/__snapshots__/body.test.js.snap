// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Body /> Rendering should render the Body 1`] = `
<Fragment>
  <div
    className="wrapper"
  >
    <Connect(Header) />
    <Connect(Content) />
  </div>
  <Footer />
</Fragment>
`;
